consensus/istanbul/backend/api.go

```diff
@@ -150,9 +150,21 @@ func (api *API) GetValEnodeTable() (map[string]*vet.ValEnodeEntryInfo, error) {
 
 // GetCurrentRoundState retrieves the current IBFT RoundState
 func (api *API) GetCurrentRoundState() (*core.RoundStateSummary, error) {
+	if !api.istanbul.coreStarted {
+		return nil, istanbul.ErrStoppedEngine
+	}
 	return api.istanbul.core.CurrentRoundState().Summary(), nil
 }
 
+// GetCurrentRoundState retrieves the current IBFT RoundState
+func (api *API) ForceRoundChange() (bool, error) {
+	if !api.istanbul.coreStarted {
+		return false, istanbul.ErrStoppedEngine
+	}
+	api.istanbul.core.ForceRoundChange()
```

Comment:
What happens when this is called while the engine is already waiting for the next round?

Reply:
it will be as though that timeout got sped up, and we then move to the next round.